include/qpdf/QPDFCryptoProvider.hh

```diff
@@ -49,7 +48,7 @@ class QPDFCryptoProvider
     // Register the given type (T) as a crypto implementation. T must be derived from QPDFCryptoImpl
     // and must have a constructor that takes no arguments.
     template <typename T>
-    QPDF_DLL static void registerImpl(std::string const& name);
+    static void registerImpl(std::string const& name);
 
```

Comment:
This one doesn't make sense to me. Unless I am missing something, this should be either private or the template  should be exposed.

Reply:
You're right. This doesn't work. And here I've been thinking qpdf supported pluggable crypto. I will investigate.